woocommerce/woocommerce
# Create variations block with empty state

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #38881



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure `Try the new product editor (Beta)` is turned on
3. Create a product without attributes and visit `/wp-admin/admin.php?page=wc-admin&path=/product/{productId}&tab=variations` replacing the `{productId}` by your recently created product.
4. You should see this screen 
<img width="899" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/7a2d57f1-e161-4bc6-9308-f5753604b9b2">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
